packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1501,8 +1501,10 @@ export interface CustomCallWithChatControlButtonProps extends CustomControlButto
 
 // @beta
 export interface CustomControlButtonProps {
+    ariaDescription?: string;
+    ariaLabel?: string;
     disabled?: boolean;
-    // (undocumented)
+    key?: string | number;
```

Comment:
Is key sufficient if Contoso wants to provide a tooltip or flyout? From trying the [fluent callout](https://developer.microsoft.com/en-us/fluentui#/controls/web/callout) in the hackathon I needed to use `id`

Reply:
(note this could be a seperate bug I am thinking of)

Reply:
Thanks for the comment James! 
Clarifying that id was an issue was definitely helpful. Updated to add id as well! :)